doc/api/cli.md

```diff
@@ -603,6 +630,36 @@ Path to the file used to store the persistent REPL history. The default path is
 `~/.node_repl_history`, which is overridden by this variable. Setting the value
 to an empty string (`''` or `' '`) disables persistent REPL history.
 
+### `NODE_UNHANDLED_REJECTION=state`
+<!-- YAML
+added: REPLACEME
+-->
+
+Setting this environment variable allows fine grained control over the behavior
+of unhandled rejections. This may be set to either one of `SILENT`, `WARN`,
+`ERROR_ON_GC`, `ERROR` or `STRICT` while the default behavior without using the
```

Comment:
just a nit... make the values lower-case. They tend to be easier for users. `--unhandled-rejections=error_on_gc`

Reply:
They will be accepted in all cases but it should be easier to read them being upper case.